boot/boot.go

```diff
@@ -368,3 +368,35 @@ func SetRecoveryBootSystemAndMode(dev Device, systemLabel, mode string) error {
 	}
 	return bl.SetBootVars(m)
 }
+
+// UpdateManagedBootConfig updates managed boot config assets if those are
+// present. Returns true when an update was carried out.
+func UpdateManagedBootConfig(dev Device) (updated bool, err error) {
+	if !dev.HasModeenv() {
+		// only UC20 devices use managed boot config
+		return false, ErrUnsupportedSystemMode
+	}
+
+	opts := &bootloader.Options{
+		// looking at root of boot partition
+		NoSlashBoot: true,
+	}
+	bl, err := bootloader.Find(InitramfsUbuntuBootDir, opts)
```

Comment:
something is off here, the dir name looks wrong, we are not in the initramfs anymore, why can't we use /boot ?

Reply:
Good point, switched to `bootloader.Find("", nil)` for the main bootloader